test/e2e/prometheus_test.go

```diff
@@ -36,6 +36,8 @@ const monitoringNamespace string = "monitoring"
 var antreaAgentMetrics = []string{
 	"antrea_agent_local_pod_count",
 	"antrea_agent_ovs_flow_table",
+	"antrea_agent_ingress_networkpolicy_rule",
```

Comment:
I had that list sorted, IDK if that has any importance though :)

Reply:
Thank you for the comment, I'll change based on your suggestion.